modules/sync/admin-sync.php

```diff
@@ -230,4 +245,41 @@ public function __call( $func, $args ) {
 			E_USER_ERROR
 		);
 	}
+
+	/**
+	 * Checks the context, and returns the value of `from_post_id` and `new_lang`.
+	 *
+	 * @since 3.7
+	 *
+	 * @param array $post_data A post array.
+	 * @return array
+	 *
+	 * @phpstan-return array{}|array{from_post_id: int<0,max>, new_lang: string}
+	 */
+	private function check_context( array $post_data ): array {
```

Comment:
If we keep this method name, then it should return a `WP_Error|array` IMHO.
Otherwise we should rename it `get_data_from_request()` or something. What do you think?

Reply:
OK why not, I renamed in [8f152ac](https://github.com/polylang/polylang/pull/1532/commits/8f152acd9e6556e79d3473247637a6717e892465).